bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/handler/NetworkHandler.java

```diff
@@ -187,6 +199,8 @@ void initialize(PresenceDetection presenceDetection) {
         presenceDetection.setRefreshInterval(handlerConfiguration.refreshInterval.longValue());
         presenceDetection.setTimeout(handlerConfiguration.timeout.intValue());
 
+        wakeOnLanPacketSender = new WakeOnLanPacketSender(handlerConfiguration.macAddress);
```

Comment:
When no mac address is configured maybe it would be nice to give a specific error message to the user in case a call is made to the actions method `sendWakeOnLanPacket`? So state in the `IllegalStateException` that the macAddress is not configured?